.github/ISSUE_TEMPLATE/module_proposal_bicep_child.yml

```diff
@@ -0,0 +1,228 @@
+name: New Bicep Child Module Proposal 🗒️
```

Comment:
Should we explicitly mention somewhere that we are only targeting resource modules?

Reply:
I wouldn't add that to the title itself, but I just updated the description to have it reflected (note how the description shows right below the title on the UI).